cruise-control/src/yaml/endpoints/addBroker.yaml

```diff
@@ -0,0 +1,184 @@
+AddBrokerEndpoint:
+  post:
+    operationId: addBroker
+    summary: Move partitions to the specified brokers, according to the specified goals
```

Comment:
Can we ensure all the `summary` and `description` fields end with dot for the new `yaml` files added in this PR.